.changeset/tabs-base-classes.md

```diff
@@ -0,0 +1,32 @@
+---
+"@patternfly/elements": major
+---
+
+### Breaking Changes
+
+- ✨ Added `TabsController.ts`
+- ❌ Removed `BaseTabs.ts`
+- ❌ Removed `BaseTab.ts`
+- ❌ Removed `BaseTabPanel.ts`
+- ❌ Removed `BaseTabs.css`
+- ❌ Removed `BaseTab.css`
+- ❌ Removed `BaseTabPanel.css`
+
+- 🔥 Tabs base classes have been removed.  These class files are no longer exported
+by the project. `<pf-tabs>` implements the new `TabsController`. Downstream
+consumers will need to update their code to use the new `TabsController` class, and 
+other inherited methods.
+- 🔥 TabExpandEvent has moved from `BaseTabs` to `TabsController`.  Downstream
+consumers will need to update their code to reimplement the import.
+- 🔥 Tabs base CSS files have been removed. The CSS files are no longer exported by
+the project. The CSS rules contained have moved into the Patternfly Elements
+implementation files `pf-tabs.css`, `pf-tab.css` and `pf-tab-panel.css` Downstream
+consumers will need to update their code to implement these base styles if needed.
+- 🔥 Tabs overflow: `OverflowController` implementation has moved from the `BaseTabs` 
+to the implemented tabs class: `pf-tabs.ts`. Downstream consumers will need to update 
+their code to reimplement overflow features.  
+
+Please review the [Tabs 2.4 to 3.0 migration guide](https://patternflyelements.org/migration/3.0/tabs) for more 
+information.
```

Comment:
Took a stab a general mock of the changeset, this still needs work to improve.  @bennypowers assuming for 3.0 we'll want a standard format for the breaking changes.  Humbly requesting guidance and help.

Reply:
Leaving this open for discussion on "migration guide" idea

Reply:
Removed the migration guide idea from the changeset.